### DISCUSSION TITLE
Open source software course survey

### DISCUSSION THREAD

**Discussion by jfigui** [openradar/erad2022#74] [General]
Hi all,

I put together a survey for the open course:
https://fr.surveymonkey.com/r/K7SQK7D

Have a look at it and feel free to suggest changes. We should not spend too much time before sending the link to the participants so I suggest that you comment on it until Friday 16 September and then on Monday 19 September @wolfidan sends it.

**Comment by wolfidan** [comment]
Hi @jfigui ,

I find it great, thanks a lot for the initiative. I had asked Alexis if they planned to do something like at ERAD level and told me it would be great, but considering how busy they all are, we'd better do it on our own... ;)

**Comment by jfigui** [comment]
@scollis @DanielMichelson @mgrover1 @kmuehlbauer @mike-dixon , any comments on that?

I think we should send the survey this week, maximum the next. If it takes to long to send it stops making sense.

Cheers,


Jordi

**Reply by kmuehlbauer** [reply]
@jfigui Sorry for not responding. The survey looks good to me.

**Reply by mgrover1** [reply]
@jfigui looks good to go!

**Reply by DanielMichelson** [reply]
Sorry for being late to the party. Thanks @jfigui for preparing the survey! It looks great. I would suggest two minor tweaks:
Question 3 should have an "Other" choice, or combined it with the "Comments" box.
Question 7: first response could read "Keep using the same tools I used before taking the course"
